docs/pages/get-started/faq.md

```diff
@@ -384,109 +409,150 @@ Clients communicate with XMTP nodes through a gRPC (or JSON/HTTP) message API.
+
+For example, a first-time user of XMTP first signs with a wallet signature to claim an inbox ID. They also provide a wallet signature to certify that the app-generated installation key is allowed to act on behalf of the inbox ID and can therefore be added to the inbox log's set of authorized keys. These signatures must be provided for each new app instance, or installation.
+
+To learn more about the signatures required for XMTP v3, see [How does XMTP establish a secure and fraud-proof relationship between identities?](#how-does-xmtp-establish-a-secure-and-fraud-proof-relationship-between-identities).
+
 ## Storage
 
 ### Where are XMTP messages stored?
 
-XMTP stores messages in the XMTP network before and after retrieval. Application-specific message storage policies may vary.
+XMTP stores messages in the XMTP network before and after retrieval. App-specific message storage policies may vary.
 
 ### What are the XMTP message retention policies?
 
-XMTP provides both `production` and `dev` network environments to support the development phases of your project.
+#### For XMTP’s blockchain and node databases:
+
+Currently, encrypted payloads are stored indefinitely.
 
-The `production` network is configured to store messages indefinitely.
+In the coming year, a retention policy is likely to be added. 
 
-XMTP may occasionally delete messages and keys from the `dev` network and will provide advance notice in the [XMTP Discord community](https://discord.gg/xmtp) and [XMTP Announcements forum](https://community.xmtp.org/c/start-here/announcements/7).
+This retention policy would represent a minimum retention period, not a maximum.
 
-Different approaches to long-term message storage are currently being researched.
+For example, a retention policy may look something like the following, though specifics are subject to change:
+  - One year for messages
+  - Indefinite storage for account information and personal preferences  
+
+The team is researching a way to provide this indefinite storage and have it scale forever. 
+- If research shows that it's possible, we'll share a plan for how it will be achieved. 
+- If research shows that it isn't possible, we'll share a plan that shows how retention periods will provide a permanent solution to storage scaling.
+
+Have questions or feedback regarding message storage and retention? Post to the [XMTP Community Forums](https://community.xmtp.org/c/development/ideas/54).
+
+#### For the on-device database managed by the XMTP SDK:
+
+Messages are stored for as long as the user decides to keep them. However, encryption keys are regularly rotated.
+
+### What are the costs of XMTP message storage and retrieval?
+
+Messages are stored off-chain on the XMTP network, with all nodes currently hosted by Ephemera. Ephemera currently absorbs all message storage and retrieval costs.
+
+There are no message storage and retrieval-related fees incurred by developers for building with the XMTP SDK.
```

Comment:
Maybe add a "today". We do expect message storage fees to be arriving next year.